examples/discovery-search-app/scripts/listProjects.js

```diff
@@ -0,0 +1,24 @@
+const DiscoveryV2 = require('ibm-watson/discovery/v2');
+const setSdkUrl = require('./setSdkUrl');
+
+async function listProjects() {
+  await setSdkUrl();
+  const client = new DiscoveryV2({ version: '2021-02-01' });
+  try {
+    const { result } = await client.listProjects();
+    return result.projects
+      .map(({ project_id, name }) => {
+        return `${project_id} (${name})`;
+      })
+      .join('\n');
+  } catch (e) {
+    console.error('Error listing projects');
```

Comment:
Just an idea: if there's an error here, there's a good chance that the example app will produce errors when trying to fetch requests related to projects. We might throw in an extra message that says something like "You may still enter a project id manually, but you might want to verify that your credentials are correct. If your credentials are invalid, you may experience errors when running the example app. To update your credentials, quit and then restart this script."

It might save the user a little bit of troubleshooting time if they already know something messed up.

Reply:
Maybe that's too verbose, and sets a precedent we don't want to uphold for each error case?